vocab.py

```diff
@@ -0,0 +1,26 @@
+from typing import Sequence, List
+
+class _Vocab:
+    # cf: https://github.com/nickbhat/mogwai/blob/main/mogwai/vocab.py
```

Comment:
not sure if it's kosher to copy code from that repo to this. what are the licenses?

Reply:
BSD3 https://github.com/nickbhat/mogwai/blob/main/LICENSE

This snippet seems relatively simple so I wasn't too concerned, and adding a dependency just for an ordered list of characters seemed like overkill haha. Perhaps it makes sense though, since mogwai is the package that trained the models and therefore determines the encoding. The way we've done it in the past is to add it as a git submodule

I was adding this file in preparation to be able to test homolog sequences, but haven't actually implemented anything that uses it.

Reply:
I'd just remove the file from the PR until you use it. Given that your code already depends on mogwai, having a formal code dependency on mogwai seems sensible.